src/OrbitQt/ProfilingTargetDialog.cpp

```diff
@@ -282,9 +281,14 @@ void ProfilingTargetDialog::SetupLocalStates() {
   state_local_.assignProperty(ui_->stadiaWidget, "active", false);
   state_local_.assignProperty(ui_->loadCaptureRadioButton, "checked", false);
   state_local_.assignProperty(ui_->targetLabel, "text", "");
+  state_local_connecting_.assignProperty(ui_->localProfilingStatusMessage, "toolTip", "");
```

Comment:
Is this supposed to be **`state_local_`**`.assignProperty(ui_->localProfilingStatusMessage, "toolTip", "");` instead of `state_local_connecting_`? Or why is this in the `// STATE state_local_` group?

Reply:
Thank you for catching that! It was a mistake on my side. It was meant to be `state_local_`, but I realized its not necessary at all, since the default of property `toolTip` is empty string. Anyways, I fixed it